### DISCUSSION TITLE
Use case and performances of Linkerd

### DISCUSSION THREAD

**Discussion by dod-ia** [linkerd/linkerd2#11392] [Q&A] [answered]
Hello,

I would like to use Linkerd but I'm not sure about the utilization of the tool. Is linkerd can be used for applications that need low latency and high performances as Log Brokers or databases in HA deployment ? The impact of the sidecar is acceptable or not realistic ? HA cluster uses TLS between nodes so mTLS from Linkerd should not be significatively longer than classic TLS, isn't it ?

Thanks

**Comment by wmorgan** [comment, marked as answer]
Hi @ventaubain, yes that should not present a problem. The big latency cost of TLS (including mTLS) is at connection establishment time so assuming the protocol keeps the connection open, the cost should be amortized fairly rapidly across requests.
